src/Driver.cpp

```diff
@@ -535,3 +555,86 @@ void Driver::GTES(void)
     waitResponseOK(command);
     modeMsgManager(command);
 }
+
+// Switch to COMMAND mode.
+void Driver::switch2CMDmode(void)
```

Comment:
I would prefer the explicit switchToCommandMode/switchToDataMode. The shortcuts are hard to read for me.